tests/e2e/specs/miscellaneous/PredefinedNamespace.spec.ts

```diff
@@ -65,22 +65,23 @@ suite(`Create predefined workspace and check it ${BASE_TEST_CONSTANTS.TEST_ENVIR
 	});
 
 	// verify that just created workspace is available for the dedicated user
-	test('Validate the created workspace is present in predefined namespace', function (): void {
+	test('Validate that predefined namespace has been created', function (): void {
 		const expectedProject: string = shellExecutor.executeArbitraryShellScript('oc get projects');
 		expect(expectedProject).contains(predefinedNamespaceName);
 	});
 
 	// make sure that the generated devspace has been created in the predefined namespace
-	test('Validate the created workspace is present in predefined namespace', function (): void {
+	test('Create test DevSpace and make sure that it has been created in the predefined project', function (): void {
```

Comment:
Test is actually waiting on workspace to be available, isn't it?
If so, `Make sure that test workspace has been created in the predefined project` test name sounds more relevant, IMHO.